chalice/awsclient.py

```diff
@@ -179,6 +186,25 @@ def _is_settling_error(self, error):
             return True
         return False
 
+    def invoke_function(self, name, context=None, payload=None):
+        # type: (str, _OPT_STR, bytes) -> Dict[str, Any]
+        kwargs = {
+            'FunctionName': name,
+            'InvocationType': 'RequestResponse',
+        }
+        if context is not None:
+            kwargs['ClientContext'] = context
+        if payload is not None:
+            kwargs['Payload'] = payload
+
+        try:
+            return self._call_client_method_with_retries(
+                self._client('lambda').invoke, kwargs, max_attempts=1
```

Comment:
Is there a specific reason we need `max_retries` on this?

Reply:
Yes its a new required parameter of that method after I rebased it.